lldb/include/lldb/Target/Statistics.h

```diff
@@ -175,6 +177,82 @@ struct StatisticsOptions {
   std::optional<bool> m_include_transcript;
 };
 
+/// A class that represents statistics about a TypeSummaryProviders invocations
```

Comment:
Can you put a note here that all members of this class need to be accessible in a thread-safe manner?

Reply:
Great idea